### DISCUSSION TITLE
Flavour: Ideas for factions

### DISCUSSION THREAD

**Discussion by RadCatish** [FakeFishGames/Barotrauma#4925] [Ideas and suggestions]
Coming up in Short term plans is "Campaign Factions: The world would be a bit bland and flat if there would be no conflicting interests, so factional rivalry and cults and other weirdos will take hold of outposts, all modifying the look and feel of the locations, and offering different missions types, all impacting on your reputation elsewhere in the game world." https://trello.com/c/hCosVSHu/195-campaign-factions

I'm sure the devs already have tonnes of ideas for this, but just in case they need some inspiration, anyone got ideas?

**Comment by RadCatish** [comment]
Europa Coalition
- Shady feeling but well-paying missions: moving riot gear to "quell protests", etc
- Sub inspections

Jovian Separatists
- Pirate aesthetic, with pirate outfit available. 
- Missions to sabotage outposts 
- Smuggling missions

The Church of Husk
- Mission to bring back Calyx Eggs, if you succeed then the outpost (or adjacent ones) may have husks next time you visit
- Options to Commune with the Husk within the game - if you have a high affinity with the CoH, perhaps new crafting recipes with calyx eggs become available, which provide stat boosts... or husk
- Killing husk on missions decreases their opinion of you
- The less they like you, the more likely you are to meet hostile husk

Children of the Honkmother
- Crafting missions: given clown outfits, have to turn them into hammers (maybe you have to gather the plastic yourselves, and can take multiple legs to complete the mission?)
- Affinity slowly turns all of the lights on your ship red
- If they don't like you, clown items may cause harm to your crew

A new, research and science focused faction?

Generic
- High affinity with a faction unlocks a "stencil" item you can use with a sprayer to leave their symbol on your sub, outposts, etc
- Each faction could have benefits and deficits for having high and low opinion
- Sub classes for each faction which only unlock and become available to buy with enough positive reputation. these should have some kind of additional benefit
- Your AI crew should start saying things aligned with factions you have high reputation with

**Reply by mLuby** [reply]
_Here's the link to the existing [faction descriptions](https://barotraumagame.com/wiki/Campaign#Faction) from the wiki._

_Relatedly, here are my thoughts on [Faction Sub Size and Naming Conventions](https://docs.google.com/document/d/1tym1bE4HRSdiqTtwBoIwndDg86imIf432qhZAzsTSEo)._

Good ideas. Some people think of the Church of Husk as being scientifically advanced toward aliens and biotech, but in my mind they're more like death cultists who don't really understand what they're playing with.

I like the idea of adding a separate Alien Science institute or corporation to the mix.

### Europa Coalition (U.C.N.)
- Themes: authoritarian, aggressive, solid, mass-produced
- Affinity for non-lethal and riot control weapons.
- Common missions: monster clearing, policing, inspecting wrecks, maintaining beacons.
- Goods are moderately expensive.
- Access to nuclear weapons. (Most factions have to steal them.)
- Common job: Security Officer.

### Jovian Separatists (S.E.P.)
- Themes: rebellion/terrorism, democracy, labor rights/unions, stolen, jury-rigged
- Affinity for explosive/improvised (e.g. crowbars) weapons.
- Common missions: smuggling/blockade running, stealing/jailbreaking, sabotaging.
- Goods are cheap.
- Common job: Mechanic.

### The Church of Husk (H.U.S.)
- Themes: transhumanism, religious, organic biotechnology, uniqueness, monsters
- Affinity for chemicals/toxins, Thalamuses.
- Aversion to fire and mechanical technology (though they still use it when necessary).
- Common missions: converting followers, scavenging, capturing monsters for testing.
- Many goods are free due to being great at ocean scavenging.
- Common job: Medical doctor.
- They have developed "improved husk eggs" that make husk infection useful: ability to swim outside without a suit begins earlier, lasts longer, and infection can be cured with just one anti-husk dose regardless of medical skill.

### Children of the Honkmother (H.N.K.)
- Themes: nihilism, anarchy, anti-authoritarian, trolling, disguise/privacy.
- Affinity for booby traps, masks.
- Common missions: stealing, sabotaging, murdering, trapping.
- Common job: Assistant.
- Can pretend to be part of other factions.

### Europan Xenology Corporation (E.X.C.)
- Theme: knowledge at all cost, secretive
- Affinity for alien(-derived plasma) weapons, artifacts.
- Common missions: retrieving artifacts, exploring ruins, surveying caves.
- Goods are very expensive.
- Access to nuclear weapons.
- Most common job: Engineer.
